### PR TITLE
Mobile Fixes

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd mobile fixes / tweaks

Before:
screen shot 2018-05-16 at 3 54 51 pm

After:
screen shot 2018-05-16 at 3 55 11 pm

### CODE REVIEW

**Comment by stvstnfrd** [comment]
@caesar2164 Can you make this a full-screen screenshot?
It's hard to tell what's happening to the sidebar content in this view.

**Comment by caesar2164** [comment]
screenshot of new layout for sidebar content:
screen shot 2018-05-16 at 4 05 46 pm

File: lagunita/lms/static/sass/partials/base/custom_overrides/_mobile.scss [hunk 1]
```diff
@@ -36,3 +36,28 @@ section.container {
         }
     }
 }
+
+@media (max-width: 768px) {
```
Comment (stvstnfrd) [inline lagunita/lms/static/sass/partials/base/custom_overrides/_mobile.scss]: Can we store the pixel width in some variable that clues us in?

like `@media (max-width: $screen_width_small)` or some such.

**Comment by caesar2164** [comment]
@stvstnfrd fixed, should I squash and merge?

**Review by stvstnfrd — APPROVED** [review]
Squash and merge.

**Pull request merged**